srcpkgs/golang-migrate/template

```diff
@@ -0,0 +1,22 @@
+# Template file for 'golang-migrate'
+pkgname=golang-migrate
+version=4.17.1
+revision=1
+wrksrc="migrate-${version}"
```

Comment:
It does build as intended without that line as well, so it should be dropped.

Reply:
Sorry, I'm just seeing this comment. Thanks for the feedback, I will test it and update the pr!

Reply:
@Bnyro I've updated the pr to remove the `wrksrc` field. Please re-review this when you get a chance